src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -224,7 +221,7 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
   private def enableUserInternal(user: SamUser, samRequestContext: SamRequestContext): Future[Unit] = {
     for {
       _ <- directoryDAO.enableIdentity(user.id, samRequestContext).unsafeToFuture()
-      _ <- enableIdentityIfTosAccepted(user, samRequestContext).unsafeToFuture()
+      _ <- registrationDAO.enableIdentity(user.id, samRequestContext).unsafeToFuture()
```

Comment:
So this PR will restrict Sam APIs to users who have accepted the ToS, but not other Terra services (assuming grace period is over and tos is enabled)? ToS enforcement in other services will come when OpenDJ is fully retired?

Reply:
The plan is for services rely on sam's checks. But services must verify on their own that it is ok to do so and stop using opendj at their pace. However I believe services are mostly doing the right thing already. See the doc linked from the epic containing the jira associated to this PR